src/de/jost_net/JVerein/util/MitgliedDummy.java

```diff
@@ -0,0 +1,887 @@
+package de.jost_net.JVerein.util;
+
+import de.jost_net.JVerein.Einstellungen;
+import de.jost_net.JVerein.gui.input.GeschlechtInput;
+import de.jost_net.JVerein.keys.Zahlungsrhythmus;
+import de.jost_net.JVerein.keys.Zahlungstermin;
+import de.jost_net.JVerein.rmi.Adresstyp;
+import de.jost_net.JVerein.rmi.Beitragsgruppe;
+import de.jost_net.JVerein.rmi.Mitglied;
+import de.jost_net.JVerein.rmi.Mitgliedfoto;
+import de.jost_net.JVerein.server.AdresstypImpl;
+import de.willuhn.datasource.GenericObject;
+import de.willuhn.datasource.rmi.DBIterator;
+import de.willuhn.datasource.rmi.DBObject;
+import de.willuhn.datasource.rmi.Listener;
+import de.willuhn.util.ApplicationException;
+
+import java.rmi.RemoteException;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+public final class MitgliedDummy implements Mitglied {
```

Comment:
Ich finde es gut das dummyMitglied in eine eigene Klasse auszulagern, allerdings würde ich eine Funktion machen die ein Miglied zurückliefert und nicht eine neu implemmentation von Mitglied erstellen, sonst sind hier die ganzen ungenutzten Funktionen mit enthalten. Also so wie der Cod bisher auch war, nur in einer extra Klasse.

Reply:
Ich hätte halt dadurch die Dummy-Werte komplett vom Datenbank-Service getrennt. Dadurch kann bspw. verhindert werden, dass durch eine falsche Verwendung des Dummys dieser in der Datenbank gespeichert wird.

Reply:
Das wäre natürlich auch möglich. Ich habe diese Form gewählt, weil so die Dummy Implementierung komplett entkoppelt vom AbstractDBObject ist